package.json

```diff
@@ -80,15 +78,16 @@
   },
   "devDependencies": {
     "ansi-colors": "1.1.0",
+    "babel-jest": "22.4.1",
```

Comment:
This will need to be moved to `dependencies`.

Reply:
How come? It's a dependency of `jest`, which is in `devDependencies` too.

Reply:
yeah, you are right.